projects/sw-routing/build.js

```diff
@@ -65,6 +70,37 @@ const bundledJSNext = () => {
   });
 };
 
-module.exports = () => {
+/** module.exports = () => {
   return Promise.all([minifiedUMD(), bundledJSNext()]);
+};**/
+
+module.exports = () => {
+  console.log('Here :S');
+  return new Promise((resolve, reject) => {
+    gulp.src([
+      path.join(__dirname, 'src', '**', '*.js'),
+      path.join(__dirname, '..', '..', 'lib', '**', '*.js')
+    ])
+    .pipe(sourcemaps.init())
+    // transform the files here.
+    .pipe(rollup({
+      entry: path.join(__dirname, 'src', 'index.js')
+    }))
+    .pipe(babel({
+      plugins: ['remove-comments'],
+      presets: ['babili']
+    }))
+    .pipe(sourcemaps.write())
```

Comment:
So by default babili doesn't remove the comment so had to add remove-comments to strip out the jsdoc and multiple licence headers.

Will look at the gulp licence task to add it back in.